packages/ketcher-core/src/application/render/restruct/reatom.ts

```diff
@@ -101,6 +101,19 @@ class ReAtom extends ReObject {
     return new Box2Abs(this.a.pp, this.a.pp)
   }
 
+  getVBoxObjOfAttachmentPoint(render: Render): Box2Abs | null {
```

Comment:
May be we can draw an image for this function?
Is `this.a.pp.add(directionVector)` shifting atom position by vector?